static/js/containers/PostPage.js

```diff
@@ -286,9 +296,10 @@ class PostPage extends React.Component<*, void> {
           title="Delete Post"
           submitText="Yes, Delete"
         >
-          Are you sure you want to delete this post?
+            Are you sure you want to delete this post?
         </Dialog>
         <Card>
+          <div>NO</div>
```

Comment:
Haha I think this was meant for the dialog

Reply:
oh oops haha